chsdi/static/doc/source/api/quickstart.rst

```diff
@@ -77,8 +77,16 @@ Default version is based on ol version 3.6.0.
 
 Available versions are (which are all based on ol3 releases):
 
-- 3.6.0
-- 3.18.2
+- 3.6.0  (LV03)
+- 3.18.2 (LV03)
+- 4.3.2  (LV03)
+- 4.4.2  (LV95)
+
+.. warning::
+   Only the current version is supported. Previous versions are given as courtesy, and may be linked to layers and resources which may not be supported anymore or even no longer exist. We strongly encourage you to migrate
+   to the latest version, currently **4.4.2**. The good practice is to **not** using the **version** parameter.
```

Comment:
It's hard to tell them 'do not use the version parameter' and in the same time ' use the last version'  which it will always use  the version parameter.